### PR TITLE
feat(net): add "helm.sh/..." annotation on lb deployment

### PR DESCRIPTION
**Pull request by hanxueluo**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hicaistar @whalecold 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hanxueluo** [comment]
/hold
让我再想一会儿，合入 release-0.5，还是开浙商 oem 分支。

File: pkg/util/lb/deployment.go [hunk 1]
```diff
@@ -23,3 +23,18 @@ func MergeDeployment(dst, src *appsv1.Deployment) (*appsv1.Deployment, bool) {
 	})
 	return dstcopy, !equal
 }
+
+func getHelmAnnotation(ann map[string]string, namespace, name string) map[string]string {
```
Comment (whalecold) [inline pkg/util/lb/deployment.go]: getHelmAnnotation 换成 mergeHelmAnnotation 是不是比较好。

Comment (hanxueluo) [reply pkg/util/lb/deployment.go]: merge 像是2个相同的对象在合并。我改成 ‘completeHelmAnnotation’ 了。

**Comment by hanxueluo** [comment]
代码合入 master，所以关闭该 PR。新PR见 #123

**Pull request closed without merging**